### PR TITLE
Add docs on how to run YAPF on itself

### PR DESCRIPTION
**Pull request by sbc100**
Ideally we should also provide a tools (or docs) on how
to run YAPF on the just the parts of the file changes
on a git branch (a la chromium's 'git cl format').

### CODE REVIEW

**Comment by hayd** [comment]
> run YAPF on the just the parts of the file changes
on a git branch

This is what [pep8radius](https://github.com/hayd/pep8radius) does fwiw, `pep8radius --yapf`.

File: HACKING.rst [hunk 1]
```diff
@@ -1,3 +1,14 @@
+Running YAPF on itself
+----------------------
+
+The run YAPF on all of YAPF::
```
Comment (pquentin) [inline HACKING.rst]: As a non-native English speaker, I don't understand what this means. Without 'The', however, it makes sense to me.

File: HACKING.rst [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+
+ $ PYTHONPATH=$PWD/yapf python -m yapf -i -r .
+
+The run YAPF on just the files changed in in the curent git branch::
```
Comment (pquentin) [inline HACKING.rst]: s/in in/in/ ?
(Same issue with 'The')

**Comment by eliben** [comment]
@pquentin I fixed these errors after pulling in the PR. Thanks for the review.

**Pull request merged**